custom.css

```diff
@@ -468,6 +537,19 @@ textarea:focus {
     margin-right: 20px;
   }
 
+  .mobile-cat {
+    font-size: 15px;
+  }
+
+  .model-button {
+    font-size: 12px;
+  }
+
+  .model-body {
+    width: 680px;
+    height: 600px;
+  }
```

Comment:
- [OPTIONAL] Just as I pointed out above, please make adjust meant to this width too before moving to the next milestone. - You can fix this by changing the with from `width: 680px` to `width: 90%`. :smile: :small_airplane: